envision/server.py

```diff
@@ -259,7 +265,7 @@ def _calculate_frame_delay(self, frame_ptr):
         return 0.5 * self._fixed_timestep_sec if not frame_ptr.next_ else 0
 
     def _wait_for_next_frame(self, frame_ptr):
-        FRAME_BATCH_SIZE = 100  # limit the batch size for bandwidth and to allow breaks for seeks to be handled
+        FRAME_BATCH_SIZE = 10  # limit the batch size for bandwidth and to allow breaks for seeks to be handled
```

Comment:
Should this be configurable or is this a magic number?

Reply:
I have made it configurable.